upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.12.yaml.template

```diff
@@ -242,7 +242,7 @@ data:
 
   # TODO: Do we want to configure this?
```

Comment:
Worth removing this comment now it's enabled I think.

Reply:
@gjtempleton  yep, I have removed it now.